### PR TITLE
Support `yarn workspace **`

### PR DESCRIPTION
**Pull request by himanoa**
See. https://classic.yarnpkg.com/en/docs/cli/workspace

### CODE REVIEW

File: src/completers/yarn.zsh [hunk 1]
```diff
@@ -1,10 +1,38 @@
...
+    local fzf_options=$1
+    shift
+
+    local parent_package=$(dirname -- $(npm root))/package.json
+    if [[ ! -f $parent_package ]]; then
+        return
+    fi
+
+    IFS=$'\0'
+    local workspace_packages_patterns=(`jq '.workspaces|map(. + "/package.json")|join("\u0000")' -r < "$parent_package"`)
```
Comment (chitoku-k) [inline src/completers/yarn.zsh:27]: Prefer not to set `IFS`.
```suggestion
    local workspace_packages_patterns=$(jq -r '.workspaces | map(. + "/package.json") | join("\u0000")' "$parent_package")
```

File: src/completers/yarn.zsh [hunk 2]
```diff
@@ -1,10 +1,38 @@
...
+
+    local package_names=()
+    for pattern in $workspace_packages_patterns; do
+        for p in ${~pattern}; do
+            package_names+=(`jq -r '.name' < $p`)
+        done
+    done
+
+    _fzf_complete --ansi --read0 --tiebreak=index ${(Q)${(Z+n+)fzf_options}} -- $@ < <(echo ${(j:\0:)package_names})
```
Comment (chitoku-k) [inline src/completers/yarn.zsh:36]: Much easier than it seems!
```suggestion
    _fzf_complete --ansi --tiebreak=index ${(Q)${(Z+n+)fzf_options}} -- $@ < <(jq -r '.name' ${~${(0)workspace_packages_patterns}})
```

File: tests/_support/yarn/workspace/package.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "workspace",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "workspaces":  ["packages/*", "examples/*"]
+
```
Comment (chitoku-k) [inline tests/_support/yarn/workspace/package.json:7]:
```suggestion
```

File: tests/yarn.zunit [hunk 4]
```diff
@@ -92,3 +92,30 @@
...
+        assert ${#actual1} equals 3
+        assert ${actual1[1]} same_as 'package/a'
+        assert ${actual1[2]} same_as 'package/b'
+        assert ${actual1[3]} same_as 'examples/a'
+    }
+
+    prefix=
+    _fzf_complete_yarn 'yarn workspace '
+}
+
```
Comment (chitoku-k) [inline tests/yarn.zunit:121]:
```suggestion
```

File: tests/_support/yarn/workspace/package.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "workspace",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "workspaces":  ["packages/*", "examples/*"]
```
Comment (chitoku-k) [inline tests/_support/yarn/workspace/package.json:6]:
```suggestion
  "workspaces": ["packages/*", "examples/*"]
```

**Review by chitoku-k — APPROVED** [review]

**Comment by chitoku-k** [comment]
Great work!

**Comment by chitoku-k** [comment]
Could you also update README so a new subcommand would appear?

**Comment by himanoa** [comment]
OK. I will write it

**Pull request merged**